### DISCUSSION TITLE
Allow file upload pull requests for non-push users

### DISCUSSION THREAD

**Discussion by reesericci** [community/community#22146] [New to GitHub]
<p>I want to be able to allow people who do not have push access to upload a file and submit it as a pull request. Is this possible?</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi there! :wave: Welcome to the Community!</p>
<p>The usual way to accomplish this would be to have the person make a fork, upload their file there, and then open a pull request!</p>
<p><a href="https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork" class="onebox" target="_blank">https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork</a></p>
